src/hearings/components/hearing-judge-names-list/hearing-judge-names-list.component.scss

```diff
@@ -1,11 +1,20 @@
-:host ::ng-deep .govuk-form-group {
-  .govuk-hint {
-    font-weight: bold;
-    color: #0b0c0c;
-    margin-bottom: 15px;
+:host ::ng-deep .exclude-person{
+  .govuk-form-group {
+    .govuk-hint {
+      font-weight: bold;
+      color: #0b0c0c;
+      margin-bottom: 15px;
+    }
+    .govuk-input {
+      width: 98% !important;
+    }
   }
-  .govuk-input {
-    width: 98% !important;
+  &.column-view{
+    .govuk-form-group {
+      .govuk-input {
+        width: 80% !important;
```

Comment:
Why are we using important

Reply:
important is used inside the lib. So, to overwrite library change used important.